crates/client/sync/src/eth/src/config.rs

```diff
@@ -0,0 +1,89 @@
+
+use std::fs::File;
+use std::path::PathBuf;
+use alloy::primitives::{
+    Address
+};
+use serde::{Deserialize, Serialize};
+
+use crate::error::Error;
+
+/// Default Anvil local endpoint
+pub const DEFAULT_RPC_ENDPOINT: &str = "http://127.0.0.1:8545";
+/// Default Anvil chain ID
+pub const DEFAULT_CHAIN_ID: u64 = 31337;
+/// Default private key derived from starting Anvil as follows:
+/// anvil -b 5 --config-out $BUILD_DIR/anvil.json
+/// PRE_PRIVATE=$(jq -r '.private_keys[0]' $BUILD_DIR/anvil.json)
+pub const DEFAULT_PRIVATE_KEY: &str = "ac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80";
```

Comment:
what does that private key correspond to? a wallet?
why would there be a default one?

Reply:
we won't be using any private key/wallet